config/config-builder/src/main.rs

```diff
@@ -90,6 +92,43 @@ struct FullNodeArgs {
     template: Option<PathBuf>,
 }
 
+#[derive(Debug, StructOpt)]
+struct KeyManagerArgs {
+    #[structopt(short = "d", long, parse(from_os_str))]
+    /// The data directory for the configs (e.g. /opt/libra/etc).
+    data_dir: PathBuf,
+    #[structopt(long)]
+    /// Specifies the JSON RPC endpoint for the key manager to communicate with.
+    json_rpc_endpoint: Option<String>,
+    #[structopt(long)]
+    /// Specifies the rotation period for key rotations (in seconds).
+    rotation_period_secs: Option<u64>,
+    #[structopt(long)]
+    /// Specifies the length of time the key manager will periodically sleep (in seconds).
+    sleep_period_secs: Option<u64>,
+    #[structopt(long)]
+    /// Specifies the type of backend to use for the key manager: in-memory, on-disk, or vault.
+    storage_backend: Option<String>,
+    #[structopt(long)]
+    /// Specifies the host URL for secure storages hosted on remote URLs.
+    storage_host: Option<String>,
+    #[structopt(long)]
+    /// Specifies the token for secure storages that use credentials.
+    storage_token: Option<String>,
+    #[structopt(long)]
+    /// Specifies a unique namespace for the secure storage.
+    storage_namespace: Option<String>,
+    #[structopt(short = "o", long, parse(from_os_str))]
+    /// The output directory.
+    output_dir: PathBuf,
+    #[structopt(short = "t", long, parse(from_os_str))]
```

Comment:
I kinda regret having short options in this app, fwiw.

Reply:
I agree -- I've removed them from the key manager options, but I don't want to remove them from others (in case I break any 3rd party deps, e.g., the containers)